Makefile

```diff
@@ -8,7 +8,7 @@ IMG=${DOCKER_NAME}:${VERSION}
 LATEST=${DOCKER_NAME}:latest
 BATS=github.com/sstephenson/bats
 VINYLDNS_REPO=github.com/vinyldns/vinyldns
-VINYLDNS_VERSION=0.9.3
+VINYLDNS_VERSION=0.9.6
```

Comment:
doesn't matter a whole lot, but since we just released 0.9.7 probably should go there

Reply:
Got it. Updated and tested with `0.9.7`.